server/channels/api4/system_test.go

```diff
@@ -83,12 +83,38 @@ func TestGetPing(t *testing.T) {
 		require.Contains(t, respString, "testvalue")
 	}, "ping feature flag test")
 
+	t.Run("ping root_status test", func(t *testing.T) {
+		if os.Geteuid() != 0 {
+			t.Skip("Skipping due to system user not having root privileges")
+		}
+		resp, err := th.SystemAdminClient.DoAPIGet(context.Background(), "/system/ping?get_server_status=true", "")
```

Comment:
It would be nice, if this test and the others use the `GetPingWithOptions` method instead of `DoAPIGet`.